ALUM/ALUM/Components/TextInputFieldComponent.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  TextInputFieldView.swift
+//  ALUM
+//
+//  Created by Harsh Gurnani and Sidhant Rohatgi on 1/18/23.
+//
+
```

Comment:
You could possibly make the password input and text input one component and change the component based on a state variable that is entered as parameter when creating an instance of the component. Do you think this would increase ease of access?